examples/Cassie/run_osc_standing_controller.cc

```diff
@@ -56,81 +57,100 @@ DEFINE_double(height, .89, "The desired height (m)");
 int DoMain(int argc, char* argv[]) {
   gflags::ParseCommandLineFlags(&argc, &argv, true);
 
+  // Build Cassie MBP
+  drake::multibody::MultibodyPlant<double> plant_w_springs(0.0);
+  addCassieMultibody(&plant_w_springs, nullptr, true /*floating base*/,
+                     "examples/Cassie/urdf/cassie_v2.urdf",
+                     true /*spring model*/, false /*loop closure*/);
```

Comment:
Why is the loop closure flag set to false?

Reply:
The loop closure flag in the function creates spring/damper element to enforce the distance constraint, but I think what we need in OSC is a kinematic distance constraint